aioambient/util/climate_utils.py

```diff
@@ -0,0 +1,130 @@
+"""Climate utility functions."""
```

Comment:
I think the vast majority (entirety?) of this module can be replaced by [`meteocalc`](https://github.com/malexer/meteocalc).

Reply:
It's not quite the same calculation. It's a slightly different algorithm for both the "dew point" and "feels like" temperature. This implementation replicates what ambientweather.net reports so that the values reported by the Ambient website and the HA integration will match exactly.